nixos/modules/virtualisation/containers.nix

```diff
@@ -134,6 +134,9 @@ let
         --setenv HOST_ADDRESS6="$HOST_ADDRESS6" \
         --setenv LOCAL_ADDRESS6="$LOCAL_ADDRESS6" \
         --setenv PATH="$PATH" \
+        ${if cfg . "additionalCapabilities" or null != null then
+          ''--capability="${concatStringsSep " " cfg . "additionalCapabilities"}"'' else ""
```

Comment:
The problem is that `additionalCapabilities` is not declared at this point and thus `optionalString` fails with "error: attribute ‘additionalCapabilities’ missing":
```
$ cat containers.nix
...
        --setenv PATH="$PATH" \
        ${optionalString (cfg . "additionalCapabilities")
          ''--capability="${concatStringsSep " " cfg . "additionalCapabilities"}" \''
        }
        ${containerInit cfg} "''${SYSTEM_PATH:-/nix/var/nix/profiles/system}/init"
...
```
 Using `or null != false` seems to be a common pattern in nixpkgs.